keepassxc-browser/ui.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+// jQuery style wrapper for getElementById()
+var $ = function(id)
```

Comment:
Why only ID? Actually you could use `querySelector` which would be more jQuery-like…

Reply:
Also the syntax is different: For query you of course need to do `$('#id')`. Your current implementation requires the ID without a hash.

But generally, do you really need to do it anyway? I think using the plain JS functions is easy enough and you also seem to do so everywhere in this PR at least.

Reply:
Plain JS is the way to go. Writing a small wrapper just makes the code more readable.

Reply:
Just make $ an alias for document.querySelector.